### PR TITLE
New PyTorch Example: Object detection in Images with R-CNN

### PR DESCRIPTION
**Pull request by ArkinDharawat**
Hey,

I have added a new PyTorch example that uses a pretrained Faster R-CNN for object detection. The labels returned are from the coco dataset.  I use the same test image that's used in the image-classification example. 

closes #<issue ID>

---

checklist:

- [ ] run `make test` and `make lint`
- [ ] test end to end manually (i.e. build/push all images, restart operator, and run `cortex deploy --refresh`)
- [ ] update the examples
- [ ] update the docs and add any new files to `summary.md`
- [ ] merge to master
- [ ] cherry-pick into release branches if applicable
- [ ] check that the docs are up-to-date in [gitbook](https://www.cortex.dev/v/master)
- [ ] alert the dev team if the dev environment changed
- [ ] delete the branch once it's merged

### CODE REVIEW

**Comment by ArkinDharawat** [comment]
Hey David,

Thanks! However, I cannot merge since it says I do not have "write access" to the repository

**Comment by deliahu** [comment]
Ah I see, sorry for the confusion. Merged!
